### ISSUE TITLE
Custom selection with parameter does not work

### ISSUE THREAD

**Issue by matthiaskoenig** [copasi/basico#18]
Based on the example in #17 I tried to set a custom selection to include the things I want to see in the results via

```
tc = run_time_course_with_output(['Time', '[S1]', '[S2]', 'Values[k1]'], 0, 5, 5)
```
But this results in the following error due to the `Values[k1]` in the selection.
```
--- c1 ---
ERROR:root:Error while initializing the simulation: >WARNING 2022-04-05T12:44:44<
  CCopasiTask (7): Problems compiling output.
Empty DataFrame
Columns: [Time, [S1], [S2], Values[k1]]
Index: []
```

**Comment by fbergmann** [comment]
Yes that needs  a fix, please use: 

```python

run_time_course_with_output(['Time', '[S1]', '[S2]', 'CN=Root,Model=NoName,Vector=Values[k1],Reference=Value']
    ...: , 0, 5, 5)
Out[12]:
   Time  [S1]  [S2]  Values[k1]
0   0.0   0.0   0.0         0.0
1   1.0   0.0   0.0         0.0
2   2.0   0.0   0.0         0.0
3   3.0   0.0   0.0         0.0
4   4.0   0.0   0.0         0.0
5   5.0   0.0   0.0         0.0

```

until the patch, sorry for the inconvenience.

**Comment by matthiaskoenig** [comment]
Thanks. Workaround works.

**Comment by fbergmann** [comment]
Released 0.17 so that the original version will work too.

**Comment by matthiaskoenig** [comment]
Thanks. solved.
